infrastructure-playbooks/purge-docker-cluster.yml

```diff
@@ -437,6 +437,12 @@
     tags:
       remove_img
 
+  - name: ensure all osd_disk_prepare logs are removed
+    file:
+      path: "{{ item }}"
+      state: absent
+    with_fileglob:
+      - "{{ ceph_osd_docker_run_script_path }}/ceph-osd-prepare-*.log }}"
 
 - name: purge ceph mon cluster
```

Comment:
Typo? Extra }} should be removed

Reply:
vim strange autocompletion on copy/paste :-/